src/components/TimeZonePicker/TimezoneAwareDatePicker.tsx

```diff
@@ -17,10 +16,11 @@ interface Props {
 }
 
 const formatDateForInput = (d: Date, tz: string): string =>
-  format(utcToZonedTime(d, tz), "yyyy-MM-dd'T'HH:mm");
+  // eslint-disable-next-line lingui/no-unlocalized-strings
```

Comment:
could probably put this in the `ignoreFunctions` list in the eslint.

Reply:
I added `format` to `ignoreFunctions`, but the lint still errored for me.